client/shaders/nodes_shader/opengl_vertex.glsl

```diff
@@ -177,7 +177,7 @@ void main(void)
 	wavePos.x /= WATER_WAVE_LENGTH * 3.0;
 	wavePos.z /= WATER_WAVE_LENGTH * 2.0;
 	wavePos.z += animationTimer * WATER_WAVE_SPEED * 10.0;
-	pos.y += (snoise(wavePos) - 1.0) * WATER_WAVE_HEIGHT * 5.0;
+	pos.y += -(snoise(wavePos) - 1.0) * WATER_WAVE_HEIGHT * 5.0;
```

Comment:
-= ?

Reply:
Unfortunately this only shifts the issue from "waving water passes through the bottom" to "waving water passes throough the top", and the latter is visually even worse in some cases.
![2024-07-30-11-58-51](https://github.com/user-attachments/assets/31bb3a14-4aad-41b2-a3b1-f13a9d87c5b1)
![2024-07-30-11-59-43](https://github.com/user-attachments/assets/b9029c15-28f6-45f3-a976-e5406aa593b6)